hearpreprocess/pipeline.py

```diff
@@ -734,13 +734,14 @@ def run(self):
         # minutes or the timestamp embeddings will explode
         sample_duration = self.task_config["sample_duration"]
 
-        if (
-            "max_task_duration_by_split" in self.task_config
-            and self.split in self.task_config["max_task_duration_by_split"]
-        ):
-            max_split_duration = self.task_config["max_task_duration_by_split"][
-                self.split
-            ]
+        if "max_task_duration_by_split" in self.task_config:
+            assert set(self.task_config["max_task_duration_by_split"].keys()) <= set(
+                SPLITS
+            )
+            if self.split in self.task_config["max_task_duration_by_split"]:
+                max_split_duration = self.task_config["max_task_duration_by_split"][
+                    self.split
+                ]
```

Comment:
else what? can you add the else clause and at least describe that it's okay if it occurs?

Reply:
simplified this part of the code to remove if-else and make more explicit